modules/azure_container_app/container_app/main.tf

```diff
@@ -5,6 +5,10 @@ locals {
       name  = upper("OCEAN__INITIALIZE_PORT_RESOURCES"),
       value = var.initialize_port_resources ? "true" : "false"
     },
+    {
+      name  = upper("OCEAN__SCHEDULED_RESYNC_INTERVAL"),
+      value = tostring(var.scheduled_resync_interval)
```

Comment:
Why are we using `tostring` on each env?

Reply:
its the same reason we have `var.initialize_port_resources ? "true" : "false"`. A string value is expected